processing/src/main/java/io/druid/query/PrioritizedExecutorService.java

```diff
@@ -39,10 +41,11 @@
 import java.util.concurrent.ThreadPoolExecutor;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
+import java.util.concurrent.atomic.AtomicLong;
 
 public class PrioritizedExecutorService extends AbstractExecutorService implements ListeningExecutorService
 {
-  public static PrioritizedExecutorService create(Lifecycle lifecycle, ExecutorServiceConfig config)
+  public static PrioritizedExecutorService create(Lifecycle lifecycle, DruidProcessingConfig config)
```

Comment:
Is it possible to subclass ExecutorServiceConfig and create PrioritizedExecutorServiceConfig so we don't have to make it depend on DruidProcessingConfig. That would separate concerns of processing configs from generic prioritized executor service configs.